locations/spiders/engel_and_volkers.py

```diff
@@ -0,0 +1,32 @@
+import json
+import re
+
+import scrapy
+
+from locations.dict_parser import DictParser
+
+
+class EngelAndVolkersSpider(scrapy.Spider):
+    name = "engel_and_volkers"
+    item_attributes = {
+        "brand": "Engel&Volkers",
+        "brand_wikidata": "Q1341765",
+    }
```

Comment:
NSI has the brand "Engel & Völkers"

```suggestion
item_attributes = {"brand": "Engel & Völkers", "brand_wikidata": "Q1341765"}
```